src/Dangl.XRechnung.Examples/XRechnungExporter.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Dangl.XRechnung.Models;
+using System.IO;
+
+namespace Dangl.XRechnung.Examples
+{
+    public static class XRechnungExporter
+    {
+        public static MemoryStream ExportInvoiceToStream()
+        {
+            var invoice = new Invoice
+            {
+                InvoiceNumber = "InvoiceNumber"
+            };
+
+            var export = XRechnungWriter.GetStream(invoice);
+            return export;
```

Comment:
That's a very smol invoice😀

Let's give it a more numbery number, and add Buyer and Seller data. Also, write some values in totals and add one line item.